docs/concepts/rate-limit-nft-concept.md

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_position: 4
+---
+
+# Rate Limiting
+
+# Overview
+
+By default, all users get three free requests on Lit every 24 hours. In order to use the network beyond the rate limit, you must reserve additional capacity. This can be done using capacity credits, which allow holders to reserve a configurable number of requests (measured in requests per second) over a fixed length of time (i.e. one week).
+
+:::note
+Currently Rate Limiting is only enabled on `Habanero` and `Manzano`
+see [here](https://developer.litprotocol.com/v3/network/networks/testnet) for test networks
+see [here](https://developer.litprotocol.com/v3/network/networks/mainnet) for mainnet networks
+:::
+
+# **Processing Requests**
+
+In order to send transactions on Lit, you must first authenticate with the [Lit nodes](https://developer.litprotocol.com/v3/sdk/authentication/overview). This can be done using one of two ways:
+
+1. [Session signatures](https://developer.litprotocol.com/v3/sdk/authentication/session-sigs/intro): signatures scoped to specific capabilities or resources, designed to be ephemeral and limited in scope. (RECOMMENDED)
+2. [Auth sigs](https://developer.litprotocol.com/v3/sdk/authentication/auth-sig): a signature obtained from a user proving they own a particular a particular key (NOT RECOMMENDED)
+
+Every time you authenticate with Lit, the request context (i.e. wallet address, owned capacity credits, etc) is extracted and validated against the Rate Limiting Module to ensure capacity has not been breached.
+
+# **The Rate Limit Module (RLM)**
+
+In order to send transactions on Lit, you must first authenticate with the [Lit nodes](https://developer.litprotocol.com/v3/sdk/authentication/overview). This can be done using one of two ways:
+
+1. [Session signatures](https://developer.litprotocol.com/v3/sdk/authentication/session-sigs/intro): signatures scoped to specific capabilities or resources, designed to be ephemeral and limited in scope. (RECOMMENDED)
+2. [Auth sigs](https://developer.litprotocol.com/v3/sdk/authentication/auth-sig): a signature obtained from a user proving they own a particular a particular key (NOT RECOMMENDED)
+
+Every time you authenticate with Lit, the request context (i.e. wallet address, owned capacity credits, etc) is extracted and validated against the Rate Limiting Module to ensure capacity has not been breached.
+
+To mint a Capacity Credit NFT, you’ll need some testLITPRO tokens. These are test tokens that hold no real value and should only be used to pay for usage on Habanero. TestLITPRO tokens should only be claimed from the verified faucet, linked [here](https://faucet.litprotocol.com/).
```

Comment:
can we make sure to keep wording consistent? Here we say 'Capacity Credit NFT' and below 'Rate Limit NFT' is used

Reply:
Okay, should the wording be `Capacity Credits` or `Rate Limit NFT`
Also should we change the category bar to `Capacity Credits` or keep the `Rate Limit` wording?

Reply:
personally in favor of Capacity Credits

Reply:
for category name, i'd say 'Paying for Usage'

Reply:
updated